plusminus/plusminus.py

```diff
@@ -1109,9 +1112,12 @@ def customize(self):
         super().customize()
         self.initialize_variable("pi", math.pi)
         self.initialize_variable("π", math.pi)
-        self.initialize_variable("τ", math.pi * 2)
+        self.initialize_variable("τ", math.tau)
+        self.initialize_variable("tau", math.tau)
         self.initialize_variable("e", math.e)
-        self.initialize_variable("φ", (1 + 5 ** 0.5) / 2)
+        self.initialize_variable("φ", phi)
+        self.initialize_variable("ϕ", phi)
```

Comment:
Are capital and lower-case phi's needed?

Reply:
Yes, take a look at this:

```python
>>> "ϕ".lower()
'ϕ'
>>> _.upper()
'Φ'
>>> _.lower()
'φ'
```
Basically `φ` and `ϕ` are two lower-case characters, while `Φ` is the upper-case of those two

Reply:
Interesting, TIL there are 2 different lower case PHI unicode symbols, one is the actual lowercase, the other is a compatibility "old-style" character for mathematical use. Ok, that's great, define them both.